gcp_variant_transforms/beam_io/vcf_parser.py

```diff
@@ -22,6 +22,11 @@
 import logging
 from collections import namedtuple
 
+import os
+import tempfile
+
+from nucleus.io.python import vcf_reader as nucleus
```

Comment:
s/nucleus/nucleus_vcf_reader so that it doesn't conflict with `import nucleus` itself (in case someone wants to do that in the future).